warehouse/warehousegrpc.go

```diff
@@ -88,3 +88,15 @@ func (w *warehousegrpc) RetryWHUploads(ctx context.Context, req *proto.RetryWHUp
 	}
 	return
 }
+func (w *warehousegrpc) ValidateObjectStorage(context context.Context, request *proto.ValidateObjectStorageRequest) (response *proto.ValidateObjectStorageResponse, err error) {
+	validateObjectReq := &ValidateObjectRequestT{
+		body: request.Body,
+	}
```

Comment:
Why create another unnecessary abstraction here ? We already have the storage request, we can create a simple function which does validation and receives the params as inputs. Wdyt ?